05a-run_ica.py

```diff
@@ -67,6 +66,8 @@ def run_ica(subject, tsss=config.mf_st_duration):
                                 baseline=config.baseline,
                                 preload=True, decim=config.decim,
                                 reject=config.reject)
+    # XXX : config.reject should not be the same for ICA as here
+    # you want to keep the EOG blinks in
```

Comment:
@SophieHerbst this we need to discuss

Reply:
@agramfort the eog is removed from the reject dict in config right now, but you are right, we should make it more explicit.